contents/stacks_and_queues/code/python/stack_queue.py

```
@@ -0,0 +1,97 @@
+class Node:
+    def __init__(self, val):
+        self.value = val
+        self.next = None
+
+class Stack():
+    def __init__(self):
+        self.stack_list = None
+    
+    def push(self, val):
+        if self.stack_list == None:
+            self.stack_list=  Node(val)
+        else :
+            temp = Node(val)
+            temp.next = self.stack_list
+            self.stack_list = temp
+            
+    def pop(self):
+        if self.stack_list == None:
+            print ("Stack is Empty")
+            return None
```

You should probably just raise a `ValueError` exception instead of returning `None`, as the `Node` value can actually be `None`, and it doesn't help you programmatically differentiate between a `None` value, and an empty `Stack`

---

done